### PR TITLE
feat: make home screen default route when app starts up

### PR DESCRIPTION
**Pull request by robinheinze**
# Overview

- Make Home screen the default route (set it to `/` instead of timeline)
- Update Sidebar components and keybindings accordingly
- Also updated the "Repo" item on the settings screen to be called "GitHub Docs" as a more clear place to find docs, in addition to the home screen

# Visual

https://github.com/infinitered/reactotron/assets/6894653/a8b58d73-2834-4790-b070-d17f624626f0

CleanShot 2024-01-05 at 09 48 34@2x

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,10 +68,10 @@ On the [Releases](https://github.com/infinitered/reactotron/releases?q=reactotro
 
 ## Want to contribute? Here are some helpful reading materials:
 
-- [**Contributing**](./docs/contributing.md)
-- [**Architecture**](./docs/architecture.md)
-- [**Monorepo**](./docs/monorepo.md)
-- [**Release**](./docs/release.md)
+- [**Contributing**](./docs/contributing/index.md)
```
Comment (trevor-coleman) [inline README.md:71]: 👏🏻✨

**Review by trevor-coleman — APPROVED** [review]
https://media.giphy.com/media/RzYweIGGBBQRXem3ib/giphy.gif

**Review by carlinisaacson — APPROVED** [review]
Also approved by @trevor-coleman and @darinwilson

**Comment by joshuayoes** [comment]
@robinheinze feel free to merge this when you are ready

**Pull request merged**